retail/interactive-tutorial/RetailProducts.Samples/AddFulfillmentPlacesSample.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_add_fulfillment_places]
+
+using Google.Cloud.Retail.V2;
+using Google.LongRunning;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+/// <summary>
+/// The add fulfillment places sample class.
+/// </summary>
+public class AddFulfillmentPlacesSample
+{
+    // The request timestamp
+    private static readonly DateTime RequestTimeStamp = DateTime.Now.ToUniversalTime();
+
+    // The outdated request timestamp:
+    // RequestTimeStamp = DateTime.Now.ToUniversalTime().AddDays(-1);
+
+    /// <summary>
+    /// Get the add fulfillment palces request.
+    /// </summary>
+    /// <param name="productName">The actual name of the retail product.</param>
+    /// <returns>Add fulfillment places request.</returns>
+    private static AddFulfillmentPlacesRequest GetAddFulfillmentRequest(string productName)
+    {
+        AddFulfillmentPlacesRequest addFulfillmentRequest = new AddFulfillmentPlacesRequest
+        {
+            Product = productName,
+            Type = "pickup-in-store",
+            AddTime = Timestamp.FromDateTime(RequestTimeStamp.AddMinutes(-1)),
```

Comment:
Why are you substracting  1 minute? And actually, from the API docs it seems that it's not necessary to set the time in any of the requests? Are you sure you need to do this?

Reply:
You are totally right, I have removed the request timestamp from all the requests from all the samples, except the one in **SetInventory**, as it's needed there for some testing purposes.